src/TimePicker.purs

```diff
@@ -101,6 +104,11 @@ type Input =
   , disabled :: Boolean
   }
 
+type Interval =
+  { start :: Maybe Time
+  , end :: Maybe Time
+  }
```

Comment:
Seems like this might be easier to work with if it were an `Option.Option`.

Reply:
There is another problem on the upgrade situation. `purescript-option` doesn't seem to have a version compatible with `purescript 0.14.1` yet which might block the upgrade on `orders` too.

Reply:
Ah good to know. I'll mention that in Sprint Planning as well.